resources/Field.php

```diff
@@ -116,41 +116,51 @@ public function render_field($field)
 		// Create a css id for our field
 		$fieldId = str_replace(['[', ']'], ['-', ''], $field['name']);
 
+		$this->prepareDropdownMarkup($field, $fieldId);
+	}
+
+	protected function prepareDropdownMarkup($field, $fieldId)
```

Comment:
Ditto why break this function? I have seen some static analysis tools warn about cyclomatic complexity > 10, but I make it 7 for the original function counting the open braces.